File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int i, j;
        for(i = 0; i < nums.length; i++){
            for(j = 0; j < nums.length; j++){
                if(nums[i] + nums[j] == target && i!=j ){
                    return new int[]{Math.min(i,j), Math.max(i,j)};
                }
            }
        }
        return new int[]{};
    }
}

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-1.cpp
class Solution {
public:
    int maxProfit(vector<int>& prices) {
        int profit = 0, min_buy = prices[0]; 

        for (int price : prices) {
            profit = max(profit, price - min_buy);
            min_buy = min(min_buy, price);
        }

        return profit;
    }
};
